### ISSUE TITLE
Giveli.st > Redirect to thank you page

### ISSUE THREAD

**Issue by micamastro** [everydotorg/donate-button#191]
Once the person finishes the donation to the list in Every, then we should redirect him to a thank you page.

Here is the flow in case there are doubts: https://www.figma.com/file/rmZJy0ozTHhwxHoBsxgnlW/giveli.st-LIL-BUB?node-id=0%3A1

Probably the best option is that once the person successfully donates on Every, we do a browser redirect to: `giveli.st/campaign-slug/thank-you`

**Comment by osdiab** [comment]
I think this can already be done with the redirectUrl query parameter on the donate flow - would that be sufficient?

**Comment by julianpoma** [comment]
@osdiab yes Omar, I think that could work!

How could we try the whole donation flow ? Is there like a dummy card or something for testing purposes?

**Comment by osdiab** [comment]
You can use a URL like this with our staging tenant:

```
https://staging.every.org/ffungi/donate?frequency=ONCE&amount=25&share_info=1&no_exit=1
```

Add the `redirect_url` query param to redirect somewhere else. For the credit card you can use `4242424....` repeating until the input fills up. Hope that helps!

**Comment by julianpoma** [comment]
@osdiab 

I just tested this and I am not being redirected back. On the last step of the donation flow, you are given two options

- If its the first time donating, you are prompted to input a password or login with social media accounts, and then you are taken to a page to finish completing your profile
- If you are already logged in, you are prompted to share your donation. If I select "skip for now", I am taken back to the nonprofit profile.

This is the URL I was using

`https://staging.every.org/OurWorldInData/donate?frequency=ONCE&amount=10&no_share=1&no_exit=1&redirect_url=https%3A%2F%2Fgivelist.vercel.app%2Flilbubs10thbirthday%2Fthank-you&share_info=1`

I tried with both `redirect_url` and `redirectUrl`

I also tried with both encoded and plain text url, just in case.
